test/fixtures/transaction_fixtures.js

```diff
@@ -76,15 +66,46 @@ const buildTransactionWithDefaults = (opts = {}) => {
         line1: opts.billing_address_line1 || 'address line 1',
         line2: opts.billing_address_line2 || 'address line 2',
         postcode: opts.billing_address_postcode || 'AB1A 1AB',
-        city: opts.billing_address_city || 'GB',
-        county: opts.billing_address_county || null,
+        city: opts.billing_address_city || 'London',
+        county: opts.billing_address_county || 'Greater London',
```

Comment:
Not sure this is realistic: although county exists in parts of our model for some reason, it can’t ever be set, so it should probably always be null

Reply:
I've removed that field from the pact fixture entirely, so it won't be asserted on by the pact